tensorflow/core/ops/image_ops.cc

```diff
@@ -435,6 +435,25 @@ REGISTER_OP("DrawBoundingBoxes")
     .Output("output: T")
     .Attr("T: {float, half} = DT_FLOAT")
     .SetShapeFn([](InferenceContext* c) {
+      // The rank of the images should be 4.
```

Comment:
Please add a unit test in image_ops_test.cc

Reply:
Thanks @rmlarsen. The test cases have been added.